packages/wonder-blocks-core/src/__docs__/exports.use-render-state.stories.mdx

```diff
@@ -0,0 +1,25 @@
+import {Meta} from "@storybook/addon-docs";
+
+<Meta
+    title="Core / Exports / useRenderState()"
+    parameters={{
+        chromatic: {
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+# useRenderState()
+
+```ts
+function useRenderState(): RenderState;
+```
+
+The `useRenderState` hook will return either:
+- `RenderState.Initial` if the component is being rendered on the server **or** it's
+  the initial rehydration render on the client.
+- `RenderState.Standard` if the component renders on the client after the initial
+  rehydration.
+
+NOTE: Although the `RenderState` enum has a third state `Root`, this value is never
+returned by `useRenderState`.
```

Comment:
I included this because I thought people might be confused by seeing `Root` in the list of auto-completions for `RenderState` without it being mentioned in the docs.